inspirehep/config.py

```diff
@@ -719,3 +719,10 @@
 RT_PASSWORD = None
 BIBCATALOG_QUEUES = "Test"
 JLAB_ARXIV_CATEGORIES = []
+
+
+# URLS
+UI_SERVER_NAME = "localhost:8081"
+INSPIREHEP_URL = f"https://{UI_SERVER_NAME}"
```

Comment:
Should I use http? This RT is not working locally, and on qa and labs we have https only

Reply:
but on local will be ``localhost:8081`` why we need UI_SERVER_NAME? can we just use ``SERVER_NAME``?

Reply:
1. It has to be decided by someone, as SERVER_NAME do not have protocol. Only way to properly generate this URL is to use url_for but this views are not registered in flask app as this is not a flask view. So protocol here is set to https by random, and it has no matter as locally we do not generate RT tickets at all. 

2.  Problem is that locally SERVER_NAME is unavailable outside docker, so If someone would run `poetry run inspirehep shell` locally it will blow up as 'SERVER_NAME' is not defined. 
Other solution is to set SERVER_NAME in config.  It would be probably better solution.

Reply:
1. Is a Flask thing. You can use `PREFERRED_URL_SCHEME` to get the proper scheme.

Reply:
by default is ``http`` on dev and we override it on prod with ``https``